sos/report/__init__.py

```diff
@@ -924,16 +924,18 @@ def _set_tunables(self):
                         try:
                             plug.options[opt].set_value(opts[plugname][opt])
                             self.soslog.debug(
-                                "Set %s plugin option to %s"
-                                % (plugname, plug.options[opt])
+                                f"Set {plugname} plugin option to"
+                                f" {plug.options[opt]}"
                             )
                         except Exception as err:
                             self.soslog.error(err)
                             self._exit(1)
                     del opts[plugname]
             for plugname in opts.keys():
-                self.soslog.error('WARNING: unable to set option for disabled '
-                                  'or non-existing plugin (%s).' % (plugname))
+                self.soslog.error(
+                    'WARNING: unable to set option for disabled '
+                    'or non-existing plugin (%s).' % (plugname)
```

Comment:
Indention looks better, but not actually replacing with an f-string.

Reply:
Yep, these should be resolved after the next commit/push addressing every `%s` remaining individually. There are a rare few that might get a `# pylint: disable=`, I'll create a list of any so they can be discussed individually, as part of the PR review.